lib/rules/no-duplicate-selectors/index.js

```diff
@@ -13,7 +13,7 @@ const resolvedNestedSelector = require("postcss-resolve-nested-selector")
 const ruleName = "no-duplicate-selectors"
 
 const messages = ruleMessages(ruleName, {
-  rejected: selector => `Unexpected duplicate selector "${selector}"`,
+  rejected: (selector, firstDuplicateLine) => `Unexpected duplicate selector "${selector}" is also at line ${firstDuplicateLine}`,
```

Comment:
Good idea! How about "used" instead of "occurrence": `Unexpected duplicate selector "${selector}", first used at line ${firstDuplicateLine}`